tests/components/brother/test_diagnostics.py

```diff
@@ -0,0 +1,86 @@
+"""Test Brother diagnostics."""
+from datetime import datetime
+import json
+from unittest.mock import Mock, patch
+
+from homeassistant.util.dt import UTC
+
+from tests.common import load_fixture
+from tests.components.brother import init_integration
+from tests.components.diagnostics import get_diagnostics_for_config_entry
+
+
+async def test_entry_diagnostics(hass, hass_client):
+    """Test config entry diagnostics."""
+    entry = await init_integration(hass, skip_setup=True)
+
+    test_time = datetime(2019, 11, 11, 9, 10, 32, tzinfo=UTC)
+    with patch("brother.datetime", utcnow=Mock(return_value=test_time)), patch(
+        "brother.Brother._get_data",
```

Comment:
Isn't this already done inside `init_integration` ?

Reply:
No because `skip_setup=True` is used with `init_integration()` function to be able to patch `utcnow()`.